tests/mlmodel_sklearn/test_feature_events.py

```diff
@@ -412,3 +413,47 @@ def _test():
         return model
 
     _test()
+
+
+numpy_str_recorded_custom_events_no_value = [
+    {
+        "users": {
+            "inference_id": None,
+            "model_name": "DecisionTreeClassifier",
+            "model_version": "0.0.0",
+            "feature_name": "0",
+            "type": "str",
+        }
+    },
+    {
+        "users": {
+            "inference_id": None,
+            "model_name": "DecisionTreeClassifier",
+            "model_version": "0.0.0",
+            "feature_name": "1",
+            "type": "str",
+        }
+    },
+]
+
+
+@reset_core_stats_engine()
+@override_application_settings({"machine_learning.inference_event_value.enabled": False})
+def test_does_not_include_value_when_inference_event_value_enabled_is_false():
+    @validate_custom_events(numpy_str_recorded_custom_events_no_value)
+    @validate_custom_event_count(count=2)
```

Comment:
Just noting that I will update this test once the label event PR is merged since this test will then need to account for the label events in the overall expected count.